src/components/flow/routers/airtime/__snapshots__/AirtimeRouterForm.test.ts.snap

```diff
@@ -156,7 +156,7 @@ Array [
             ],
             "category_uuid": "763e4844-3e1b-407a-a1b5-5fdfcd308b41",
             "type": "has_category",
-            "uuid": "1e47a1e1-3c67-4df5-adf1-da542c789adb",
+            "uuid": "e67ce051-f3e4-4c42-807b-e393df5fa382",
```

Comment:
cases uuids change which would break localization.. but these types of nodes shouldn't have localized case args